### PR TITLE
Angular 10 update blog

### PR DESCRIPTION
**Pull request by swethakundaram**
Added just the basic details. Need more details about the docker image.

### CODE REVIEW

File: website/blog/2021-09-01-angular-10-update.md [hunk 1]
```diff
@@ -25,18 +25,26 @@ Please note that this Angular update does not include Ivy Engine but covers some
 
 ## How WaveMaker can Help
```
Comment (krishnaraoch) [inline website/blog/2021-09-01-angular-10-update.md:27]: //Better to consider these... rewrite as needed

This impact building WaveMaker apps outside WaveMaker using your own CI/CD system.
So we expect you to upgrade node and npm on your build infrastructure with above mentioned versions.
Or you can utilize resources given by WaveMaker Team.
WaveMaker team is publishing Dockerfile, Docker image for this.

- Docker file.
  - Have commands to install required software utilities for building WaveMaker inside the container. 
  - Instructions to Build docker image out of this file and use docker image to build WaveMaker App.
  - Link to Docker Github and Dockerhub
- Docker Image
  - PreBaked Docker Image by WaveMaker team.
  - Customers utilize this to build a WaveMaker app by passing source code as  volume while running containers out of this image.
  - Docker Image URL on Docker hub
- Docker Image with NPM and Maven dependencies.
- PreBaked Docker Image by WaveMaker team.
- Same as point:2, but with Maven and NPM dependencies are pre baked.
- So Customers can utilize this image to build WaveMaker apps without contacting Maven or NPM repositories. 
- Docker hub url

**Review by krishnaraoch — CHANGES_REQUESTED** [review]

**Comment by swethakundaram** [comment]
@krishnaraoch, I have added the requested changes in the docker contents section as a note. Please suggest if it is fine that way? If that section is needed if we are pointing users to another document.

**Review by rnaresh — APPROVED** [review]

**Review by rnaresh — APPROVED** [review]

File: website/blog/2021-09-01-angular-10-update.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+|**Node** | 10.15 | 12.22.3 |
+| **NPM**| 6.4 | 6.14.13 |
+
+:::important
+**Update Node JS Version**: Angular 10 requires Node JS version 12.22.3. So you must upgrade Node and NPM on your build infrastructure with the versions mentioned above.
+:::
+
+## How WaveMaker can Help
+
+There are two ways to upgrade to the platform with Angular 10.
```
Comment (krishnaraoch) [inline website/blog/2021-09-01-angular-10-update.md:33]: replace Platform by "your build system"

File: website/blog/2021-09-01-angular-10-update.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+1. Download and install Node and NPM in the build system.
```
Comment (krishnaraoch) [inline website/blog/2021-09-01-angular-10-update.md:35]: 1. Download and install Node and NPM in the build system as above mentioned versions.

File: website/blog/2021-09-01-angular-10-update.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+2. Use resources provided by the WaveMaker on demand. WaveMaker team will publish the Docker file and Docker image.
```
Comment (krishnaraoch) [inline website/blog/2021-09-01-angular-10-update.md:36]: Use resources provided by the WaveMaker Team.

File: website/blog/2021-09-01-angular-10-update.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+### Using Docker Option
+
+If you are using Docker images, you can use them in the following ways:
```
Comment (krishnaraoch) [inline website/blog/2021-09-01-angular-10-update.md:40]: If you are using Docker images for Building WaveMaker Application,  you can use them in the following ways:

File: website/blog/2021-09-01-angular-10-update.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+- Using them from the public repo containing pre-built Docker image developed by WaveMaker.
```
Comment (krishnaraoch) [inline website/blog/2021-09-01-angular-10-update.md:42]: developed and published

File: website/blog/2021-09-01-angular-10-update.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+- Using Docker files to build the docker images. For more information, see [Build with Docker](/learn/app-development/deployment/build-with-docker). 
+
+:::note
```
Comment (krishnaraoch) [inline website/blog/2021-09-01-angular-10-update.md:45]: Skip this

**Review by krishnaraoch — CHANGES_REQUESTED** [review]

**Review by krishnaraoch — APPROVED** [review]

**Pull request merged**